plugins/conftest.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: conftest
+spec:
+  version: "v0.5.2"
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/instrumenta/conftest/archive/v0.5.2.zip
+    head: https://github.com/instrumenta/conftest/archive/master.zip
+    sha256: "5b587c11e4b1de8679c39e07a514c68e54de57987fc3eb32dc7946e78994359a"
+    files:
+    - from: "/*/plugin/*.sh"
+      to: "."
+    bin: "kubectl-conftest.sh"
+  shortDescription: Test your Kubernetes objects using Open Policy Agent
+  homepage: https://github.com/instrumenta/conftest
+  caveats: |
+    This plugin needs the following programs:
+    * conftest
```

Comment:
Is the plugin itself a dependency?

Reply:
Nope. The plugin provides a kubectl context to `conftest`, which is a standalone application https://github.com/instrumenta/conftest

I went the shell script plugin route for speed of implentation. If folks find it interesting I'll build a separate binary that uses go-client and conftest as a library to avoid the need for the shell dependencies.